node/src/assets.rs

```diff
@@ -347,6 +348,32 @@ where
     last_id: Mutex<Option<UniqueId>>,
 }
 
+/// Iterates over a key range in column `db_col`, determined by  [`DistributedAssetStorage::<T>::make_prefix_range(my_participant_id, domain_id)`],
+/// Deletes all entries in `db_col` that evaluate `false` for `is_subset_of_active_participants(persistent_participants)`
+pub fn clean_db<T>(
```

Comment:
It feels like this could/should be a method on the `DistributedAssetStorage` type. Or is there anything preventing this?

Reply:
Ah right, because the storage is an in-memory copy of the on-disk database. Makes sense, thanks for clarifying. Although it feels weird to explicitly load all data in-memory in the first place. Afaik RocksDB will keep most relevant data in-memory already.